.gitignore

```diff
@@ -1,6 +1,6 @@
 log.*
 *.swp
 node_modules
-helios.d.ts
-helios.d.ts.map
+# helios.d.ts
```

Comment:
I don't think we need to track these auto-generated files. They are only really useful for the npm package

Reply:
I've included them for my testing, I will remove them prior to the final merge.

I'm testing the package by including it directly from the GitHub repo